lib/config/discover.ml

```diff
@@ -1,21 +1,28 @@
-open Base
-open Stdio
 module C = Configurator.V1
+module S = String
 
-let write_sexp fn sexp =
-  Out_channel.write_all fn ~data:(Sexp.to_string sexp)
+let cfig c =
+  let default: C.Pkg_config.package_conf = {
+      libs = ["-lR"; "-lRmath"];
+      cflags = [] }
+  in
+  let rlib = C.Process.run c "pkg-config" ["--libs-only-L"; "--libs-only-l"; "libR"] in
+  let rmlib = C.Process.run c "pkg-config" ["--libs-only-L"; "--libs-only-l"; "libRmath"] in
+  let rflg = C.Process.run c "pkg-config" ["--cflags"; "libR"] in
+  let rmflg = C.Process.run c "pkg-config" ["--cflags"; "libRmath"] in
+  let chk = (rlib.exit_code, rmlib.exit_code, rflg.exit_code, rmflg.exit_code) in
+  let lib = (S.split_on_char ' ' (S.trim rlib.stdout)) @ (S.split_on_char ' ' (S.trim rmlib.stdout)) in
```

Comment:
Could you add a function definition for `fun x -> S.split_on_char (S.trim x)`?

Reply:
Yup, done.